src/test/java/com/github/anba/es6draft/test262/Test262Info.java

```diff
@@ -400,12 +400,12 @@ public void setEs7id(String es7id) {
             this.es7id = es7id;
         }
 
-        public String getId() {
-            return id;
+        public String getEsid() {
+            return esid;
         }
 
-        public void setId(String id) {
-            this.id = id;
+        public void setId(String esid) {
```

Comment:
The setter method name also needs to be updated to "setEsid", otherwise snakeyaml doesn't link the setter to the "esid" yaml-property: https://travis-ci.org/anba/es6draft/jobs/111073739#L362